app/api/routes.py

```
@@ -236,15 +239,30 @@ def ocr_and_save(
+            save_to_gcs=False,
+            user_hash=user_hash
         )
 
-        # JSON → DB 저장/갱신
+        # 2. DB 저장
         rec = save_pdrecord_json(data, db, user_id=current_user.id)
 
+        # 3. GCS 업로드
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        filename = f"ocr_{rec.record_date}_{timestamp}.jpg"
+        gcs_path = upload_to_gcs(
```

Ruff (0.14.4)

Local variable `gcs_path` is assigned to but never used

Remove assignment to unused variable `gcs_path`

(F841)